src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs

```diff
@@ -25,7 +26,7 @@ namespace Files.App.Views.LayoutModes
 {
 	public sealed partial class ColumnViewBase : StandardViewBase
 	{
-		private IUserSettingsService UserSettingsService { get; } = Ioc.Default.GetRequiredService<IUserSettingsService>();
+		private readonly IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
```

Comment:
Could you revert that?

Reply:
Actually, I'll remove it completely since it's no more needed